src/screens/authentication/create_new_wallet/DoubleCheckItScreen/index.tsx

```diff
@@ -0,0 +1,107 @@
+import React, {useEffect, useState} from 'react';
+import {ScrollView, StyleSheet} from 'react-native';
+import {ScreenProps} from "navigation/ScreenProps";
+import CreateNewWalletRouter from 'navigation/CreateNewWalletNavigation/CreateNewWalletRouter';
+import _ from 'lodash';
+import {CLayout, CHeader} from "components";
+import {scale} from "device";
+import CButton2 from "components/CButton2";
+import {useNavigation} from "@react-navigation/native";
+import {CheckItem} from "screens/authentication/create_new_wallet/components";
+import {StackNavigationProp} from "@react-navigation/stack";
+
+const numberOfRandom = 8;
+
+// @ts-ignore
+const DoubleCheckItScreen: React.FC<ScreenProps<CreateNewWalletRouter.DOUBLE_CHECK_IT_SCREEN>> = ({route}) => {
+    const {data} = route.params;
+    const [listData, setListData] = useState<any>([]);
+    const [listDataSelected, setListDataSelected] = useState<any>([]);
+
+    const {navigate} = useNavigation<StackNavigationProp<any>>();
+
+    useEffect(() => {
+        const numberOfWords = Math.floor(data.length / 3);
+
+        const randomList = _.sampleSize(data, numberOfWords);
+        let restList = getArrayNotInArray(data, randomList);
+        restList = _.shuffle(restList);
+
+        const list = randomList.map((item, index) => {
+            let listWords = [{...item, isKey: true}];
+            const randomWords = _.sampleSize(restList, 2);
+            restList = getArrayNotInArray(restList, randomWords);
+            listWords = listWords.concat(randomWords);
+            listWords = _.shuffle(listWords);
+            return listWords;
+        });
+        setListData(list);
+    }, []);
```

Comment:
Pls add data as deps. We should to fix  Eslint errors if possible.
<img width="718" alt="Screen Shot 2022-01-17 at 18 04 19" src="https://user-images.githubusercontent.com/6711653/149760351-37f796e1-1fbc-4e68-8f76-e08d3783cc1a.png">